app/auth/models.py

```diff
@@ -10,8 +10,9 @@
 
 
 class JWT(object):
```

Comment:
Does this change expiration for both API and browser. I think we probably want different times for the two locations. For the API tokens should be very short as the client can renew. For browser we probably want long tokens (a month or a year).